docs/source/storeconv.md

```diff
@@ -22,11 +22,12 @@ Useful references related to this document include:
 
 ## Naming Identifiers
 
-This section states what identifiers used by an xcube data store SHALL look like.
+This section states how identifiers of an xcube data store SHALL look like.
```

Comment:
I don't find"of" appropriate here; it misleadingly implies that these identifiers only apply to stores.

"how" is ungrammatical. "How does it look?" and "What does it look like?" are correct sentences, but not "How does it look like?". But in any case I was never entirely happy with the slightly vague use of "look" here. I would suggest something like: "This section defines the structure of the identifiers used by an xcube data store".

Reply:
I agree with Pont.

Reply:
Maybe lose the "the" in Pont's sentence, otherwise I prefer his sentence.